README.md

```diff
@@ -135,51 +142,77 @@ and introduce changes to the code just then, rather than build only when it's ne
 but rather than use Chrome extension page, it's quicker to use 
 [this reloading app](https://chrome.google.com/webstore/detail/extensions-reloader/fimgfedafeadlieiabdeeaodndnlbhid).
 
-### production
-```
-npm run build-extension
-```
-Modifies `dist/browser-extension` as a side effect.
-
-A package ready to upload to Chrome Web Store is `dist/pp-chrome.zip`.
-
-## Content script (= client) build
-
+## Content script (= client-only) builds
+All builds compile to a `dist/client` directory.
 
-### client + hot reloading + dev configuration
+**~~client + hot reloading~~**
+_Out of operation for now._ 
 ```
-npm start
+npm run start-client
 ```
 
-### client + prod configuration
+**~~client compilation~~** 
+_Out of operation for now._ 
 ```
-npm run build
+npm run build-client
 ```
 
+
 ## Connecting to backend
 
 This application works with PrzypisPowszechny API server - https://github.com/PrzypisPowszechny/pp.
 
 #### Default instance
-By default it's set to connect to a public instance of the development server. 
+By default it set to connect to public development instance of the server. 
 
 #### Local instance
-If you want to connect to your local instance, add `--env.api=local` arg to the command starting client, e.g.
+If you want to connect to your local instance, add `--env.PP_API=local` arg to command starting client or just set `PP_API=local` environment var, e.g.
+```
+npm run start -- --env.PP_API=local
 ```
-npm run start-extension -- --env.api=local
+which is equivalent to
 ```
-#### Documentation of API
+PP_API=local npm run start
+```
+All development values of `PP_API`:
+ - `local` - localhost port 8000
+ - `local-alt` - localhost port 8080
```

Comment:
There's a potential clash with localhost client application, but we may change the client application port to a different one, too, when we're working on it soon.